src/mudlet.h

```diff
@@ -671,7 +673,7 @@ private slots:
     // Has default form of "en_US" but can be just an ISO langauge code e.g. "fr" for french,
     // without a country designation. Replaces xx in "mudlet_xx.qm" to provide the translation
     // file for GUI translation
-    QString mInterfaceLanguage;
+    QString mInterfaceLanguage {};
```

Comment:
`QString` s do not need initialisation like that - the default constructor constructs them as a null which is almost indistinguishable from them being empty. {Only `QString::isNull()` and a very esoteric corner case that I found a few years back but cannot remember the details of - can tell the difference!}

Reply:
True, but it doesn't hurt for consistency and this is the way to go to initialise values in the future.